cpp/tests/utilities/cudf_test_utils.cuh

```diff
@@ -46,114 +80,41 @@ void print_typed_column(col_type * col_data,
   cudaMemcpy(h_data.data(), col_data, num_rows * sizeof(col_type), cudaMemcpyDeviceToHost);
 
 
-  const size_t num_masks = gdf_get_num_chars_bitmask(num_rows);
+  const size_t num_masks = valid_size(num_rows);
   std::vector<gdf_valid_type> h_mask(num_masks);
   if(nullptr != validity_mask)
   {
-    cudaMemcpy(h_mask.data(), validity_mask, num_masks * sizeof(gdf_valid_type), cudaMemcpyDeviceToHost);
+    cudaMemcpy((int *) h_mask.data(), validity_mask, num_masks * sizeof(gdf_valid_type), cudaMemcpyDeviceToHost);
   }
 
-
-  for(size_t i = 0; i < num_rows; ++i)
-  {
-    // If the element is valid, print it's value
-    if(true == gdf_is_valid(h_mask.data(), i))
+  if (validity_mask == nullptr) {
+    for(size_t i = 0; i < num_rows; ++i)
     {
-      std::cout << h_data[i] << " ";
+      if (sizeof(col_type) == 1)
+        std::cout << (int)h_data[i] << " ";
```

Comment:
Use appropriate C++ cast instead of C-style cast.

Reply:
Done.